intera_tools_description/urdf/electric_gripper/fingers/paddle_tip.xacro

```diff
@@ -19,8 +19,8 @@
         </geometry>
       </collision>
       <inertial>
-        <origin rpy="0 0 ${pi*(g_reflect+1)/2}" xyz="0 0 0"/>
-        <mass value="0.01"/>
+        <origin rpy="0 0 0" xyz="0 0 0"/>
+        <mass value="0.0"/>
```

Comment:
No mass cal be 0.0 exactly, or Gazebo barfs with a not-super helpful message:

> [ERROR] [1503000293.597456105, 0.151000000]: This robot has a joint named "right_gripper_r_finger_joint" which is not in the gazebo model

see this [answers.ros.org answer](https://answers.ros.org/question/102276/this-robot-has-a-joint-named-foo-which-is-not-in-the-gazebo-model/?answer=102681#post-id-102681) for details